1_turb_abl_fsi/turb_run/nrel5mw_nalu.yaml

```diff
@@ -154,6 +158,12 @@ realms:
       use_wall_function: no
       turbulent_ke: 0.0
 
+  - wall_boundary_condition: bc_wall
+    target_name: tower-wall
+    wall_user_data:
+      use_wall_function: no
+      turbulent_ke: 0.0
+
```

Comment:
We need this too - 
  - symmetry_boundary_condition: bc_ground
    target_name: tower-ground
    symmetry_user_data: